Sources/NIOPosix/SelectableEventLoop.swift

```diff
@@ -66,7 +66,10 @@ internal final class SelectableEventLoop: EventLoop {
     internal var scheduledTaskCounter = NIOAtomic.makeAtomic(value: UInt64(0))
     @usableFromInline
     internal var _scheduledTasks = PriorityQueue<ScheduledTask>()
-    private var tasksCopy = ContiguousArray<() -> Void>()
+
+    // We only need the ScheduledTask's task closure. However, an `Array<() -> Void>` allocates
+    // for every appended closure. https://bugs.swift.org/browse/SR-15872
+    private var tasksCopy = ContiguousArray<ScheduledTask>()
```

Comment:
Can you add this PR to https://github.com/apple/swift-nio/blob/main/docs/workarounds.md?